retail-managment-system/rms-ui/src/app/pages/installments/installment/installment.component.scss

```diff
@@ -31,3 +31,22 @@ td {
 .add-icon {
   cursor: pointer;
 }
+.print-btn {
+  background-color: #002d40;
+  color: white;
+  width: 100px;
+  height: 50px;
```

Comment:
color: white;
  width: 100px;
  height: 50px;
  why didn't used bootstrap for this?